src/diff/props.js

```diff
@@ -39,39 +38,28 @@ const XLINK_NS = 'http://www.w3.org/1999/xlink';
  * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
  */
 function setProperty(dom, name, value, oldValue, isSvg) {
+	let v;
+
 	name = isSvg ? (name==='className' ? 'class' : name) : (name==='class' ? 'className' : name);
 
 	if (name==='style') {
+		// Always clear the previous styles
+		dom.style.cssText = EMPTY_OBJ;
```

Comment:
Elaborate, I like it :D

Reply:
It seems odd to use an Object here. It's basically doing:

```js
dom.style.cssText = '[object Object]'
```

... because the cssText setter coerces to a String.

Reply:
Gonna change it. Thanks!